api/src/main/java/org/pmiops/workbench/workspaces/resources/WorkspaceResourceMapper.java

```diff
@@ -67,6 +67,8 @@ public interface WorkspaceResourceMapper {
   @Mapping(target = "lastModifiedEpochMillis", source = "cohortReview.lastModifiedTime")
   ResourceFields fromCohortReview(CohortReview cohortReview);
 
+  // TODO combine fromConceptSet/fromDbConceptSet
```

Comment:
I needed to add a new version of fromConceptSet() which took a `DbConceptSet` for this PR, but in the longer term we should be able to choose one.